sdks/python/generate_pydoc.sh

```diff
@@ -31,43 +31,119 @@ rm -rf target/docs/*
 
 mkdir -p target/docs/source
 
-# Exclude internal/experimental files from the documentation.
+# Sphinx apidoc autodoc options
+export SPHINX_APIDOC_OPTIONS=\
+members,\
+undoc-members,\
+show-inheritance
+
+# Exclude internal/experimental/Cython paths/patterns from the documentation.
 excluded_internal_code=(
+    apache_beam/coders/stream.py*
     apache_beam/examples/
     apache_beam/internal/clients/
-    apache_beam/io/gcp/internal/clients/
+    apache_beam/io/gcp/internal/
+    apache_beam/io/gcp/tests/
     apache_beam/runners/api/
     apache_beam/runners/test/
+    apache_beam/runners/dataflow/internal/
     apache_beam/runners/portability/
     apache_beam/runners/worker/
-    apache_beam/runners/dataflow/internal/clients/
-    apache_beam/testing/data/)
-
-python $(type -p sphinx-apidoc) -f -o target/docs/source apache_beam \
-    "${excluded_internal_code[@]}" "*_test.py"
+    *_pb2.py
+    *_test.py
+    *_test_common.py
+)
 
-# Remove Cython modules from doc template; they won't load
-sed -i -e '/.. automodule:: apache_beam.coders.stream/d' \
-    target/docs/source/apache_beam.coders.rst
+python $(type -p sphinx-apidoc) -fMeT -o target/docs/source apache_beam \
+    "${excluded_internal_code[@]}"
 
-# Create the configuration and index files
+#=== conf.py ===#
```

Comment:
The previous comment `Create the configuration and index files`, is more explanatory.